contracts/meta-oracles/LinearizedPriceDataSource.sol

```diff
@@ -59,7 +61,7 @@ contract LinearizedPriceDataSource is
      */
     constructor(
         uint256 _interpolationThreshold,
-        address _medianizerAddress,
+        IMedian _medianizerAddress,
         string memory _dataDescription
     )
         public
```

Comment:
Below, you don't need `IMedian(_medianizerAddress)` if its already of `IMedian` type